OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2456,6 +2456,17 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				// 'CloseEnough' on 'RepairableNear' uses WDist now
+				if (engineVersion < 20151201)
+				{
+					if (node.Key == "RepairableNear")
+					{
+						var ce = node.Value.Nodes.FirstOrDefault(n => n.Key == "CloseEnough");
+						if (ce != null && !ce.Value.Value.Contains("c0"))
```

Comment:
You should check for just `c` in my opinion.
Someone may have rules that predate this but that look like:
```yaml
Foo:
    RepairableNear:
        CloseEnough: 5c1
```

Highly doubtful, but it doesn't hurt to be a bit more careful.